agent/directory.go

```diff
@@ -56,7 +60,7 @@ func (a *Agent) createTaskDirectory(tc *taskContext, taskDir string) (string, er
 // was executing. It does not return an error because it is executed at the end of
 // a task run, and the agent loop will start another task regardless of how this
```

Comment:
can we update the comments to mention about disabling host if cleanup fails!

Reply:
Sure, I updated the `removeTaskDirectory` comment to describe what happens when it fails to clean up the task dir.